RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Releasing a version
+
+## Prepare
+
+Ensure that your `go version` is at least 1.15.x.
```

Comment:
```suggestion
Ensure that your `go version` is not older than indicated in the header of `go.mod`.
```

`go.mod` says 1.16 today. Let's take a step towards de-duplicating the source of truth for version numbers.